integration/node/node-worker-threads.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path')
+const pmrpc = require('../../build/pm-rpc')
+const {Worker} = require('worker_threads')
```

Comment:
can you add it as unit?

Reply:
`.unit` files are taken into karma, meaning they would run with a browser environment, so no, we need the node:worker_threads instead of WebWorker